c10/util/TypeList.h

```diff
@@ -247,6 +250,22 @@ template<> struct reverse<typelist<>> final {
 template<class TypeList> using reverse_t = typename reverse<TypeList>::type;
 
 
+template<class TypeList, template<class> class Condition, class Enable = void> struct find_if final {
```

Comment:
This template isn't standardized in a later C++, is it? It should have a doc then, no?